lib/client/api.js

```diff
@@ -281,7 +286,18 @@ API.prototype.createAddress = function(cb) {
   var data = this._loadAndCheck();
 
   var url = '/v1/addresses/';
-  this._doPostRequest(url, {}, data, cb);
+  this._doPostRequest(url, {}, data, function(err, address) {
+    if (err) return cb(err);
+
+    if (data.publicKeyRing.length != data.n)
```

Comment:
I'm not sure this is correct. Shouldn't this check be performed server-side only?